cmd/cloudx/project/create.go

```diff
@@ -39,8 +39,13 @@ func NewCreateProjectCmd() *cobra.Command {
 				return err
 			}
 
-			if (len(name) == 0 || len(environment) == 0) && flagx.MustGetBool(cmd, cmdx.FlagQuiet) {
-				return errors.New("you must specify the --name and --environment flags when using --quiet")
+			wsID := h.WorkspaceID()
+			if wsID == nil {
+				return errors.New("a workspace is required to create a project")
```

Comment:
Can you hint at what the user needs to do please?

Reply:
The error returned here will be printed along with the full command help message. That should be sufficient IMO.